docs/devsetup/java/maven.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: default
+title: Maven Setup for Java
+grand_parent: Setup for Contributors
+parent: Java Setup
+---
+
+<!--
+© 2016 and later: Unicode, Inc. and others.
+License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# Maven Setup for Java
+{: .no_toc }
+
+## Contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+
+Maven is a pretty standard build tool in the Java ecosystem with a very well-defined
```

Comment:
Nitpick: remove "pretty" :-) After 20+ years it is standard :-)

Reply:
Done.